dh_collections_prism_collection_community.json

```diff
@@ -0,0 +1,1120 @@
+{
+  "a86e1412-d72c-4cae-b8ca-16fd834cb128": {
+    "collection_id": "2019-2020",
+    "community_id": "biostatistics-collaboration-center-lecture-series"
+  },
+  "fc389d13-2430-409b-82fd-a4b26613d350": {
+    "collection_id": "Fall 2016",
```

Comment:
I think we might need to remap this so that collection ids are parameterized, but that should be simple. In the mapping file there were only ids for the community and not the collection so I just took the title of the collection from DH and dropped it in. If we need to remap it we could do something like:

```
JSON.parse(mapping_json).each do |entry|
  current_collection_id = entry["collection_id"]
  new_collection_id = current_collection_id.strip_special_characters.parametrize
  entry["collection_id"] = new_collection_id
end

# write out new JSON hash
```
This obviously isn't exact, but just to say it's something we can do pretty easily if need be.

Reply:
Passing human labels like this for the collection is totally fine. The import side will handle the translation when the time comes too. 

Also, we should be able to merge/replace `collections_to_communities.json` with this file and thus have a single source of truth for dh collection to prism community/collection (and file to community/collection). Can be done separately.